Modules/_testcapimodule.c

```diff
@@ -7304,6 +7304,7 @@ PyInit__testcapi(void)
     PyModule_AddObject(m, "PY_SSIZE_T_MAX", PyLong_FromSsize_t(PY_SSIZE_T_MAX));
     PyModule_AddObject(m, "PY_SSIZE_T_MIN", PyLong_FromSsize_t(PY_SSIZE_T_MIN));
     PyModule_AddObject(m, "SIZEOF_TIME_T", PyLong_FromSsize_t(sizeof(time_t)));
+    PyModule_AddObject(m, "HEX_VERSION", PyLong_FromLong(Py_Version));
```

Comment:
Hum, in a previous version of your PR, you used Py_Version to initialize sys.hexversion. Why did you revert this change? If you use Py_Version to initialize sys.hexversion, IMO you can remove this unit test.

Reply:
cc @pablogsal who asked to add the test.

Reply:
👍

Reply:
> Hum, in a previous version of your PR, you used Py_Version to initialize sys.hexversion. Why did you revert this change? If you use Py_Version to initialize sys.hexversion, IMO you can remove this unit test.

@vstinner Hmm, I don't recall doing that, for otherwise the current test would have been redundant, as you have noted. The idea is that `sys.hexversion` is already tested for the expected format and value, so checking that `sys.hexversion` equals the value exposed by `Py_Version` should be enough.

Reply:
If you would like to keep it, please rename it to "Py_Version".